stack/rest/src/test/java/org/apache/usergrid/rest/applications/utils/TestUtils.java

```diff
@@ -44,4 +46,21 @@ public static UUID getIdFromSearchResults( JsonNode rootNode, int index ) {
 
         return UUIDUtils.tryExtractUUID( entity.get( "uuid" ).asText() );
     }
+
+    /** Get the uuid at the given index for the root node.  If it doesn't exist, null is returned */
+    public static UUID getIdFromSearchResults( Collection collection, int index ) {
```

Comment:
Since it seems like one test is using the TestUtils file, is it really worth having another file just for some helper methods?